BluetoothExplorer/Controller/GATT Characteristic Editors/DateTimeCharacteristicViewController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  DateTimeCharacteristicViewController.swift
+//  BluetoothExplorer
+//
+//  Created by Carlos Duclos on 6/25/18.
+//  Copyright © 2018 PureSwift. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import CoreData
+import Bluetooth
+import GATT
+
+final class DateTimeCharacteristicViewController: UIViewController {
+    
+    // MARK: - IB Outlets
+    
+    @IBOutlet private(set) weak var dateTimeLabel: UILabel!
+    
+    @IBOutlet private(set) weak var datePicker: UIDatePicker!
+    
+    // MARK: - Properties
+    
+    var value = GATTDateTime(date: Date())
+    
+    var valueDidChange: ((GATTDateTime) -> ())?
+    
+    var minimumDate: Date? {
+        
+        var dateComponents = DateComponents()
+        dateComponents.year = Int(GATTDateTime.Year.min.rawValue)
+        dateComponents.timeZone = TimeZone(identifier: "UTC")
+        
+        return Calendar.current.date(from: dateComponents)
+    }
+    
+    var maximumDate: Date? {
+        
+        var dateComponents = DateComponents()
+        dateComponents.year = Int(GATTDateTime.Year.max.rawValue)
+        dateComponents.timeZone = TimeZone(identifier: "UTC")
+        
+        return Calendar.current.date(from: dateComponents)
+    }
+    
+    // MARK: - Loading
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        guard let minimumDate = minimumDate
+            else { assertionFailure("Couldn't create minimumDate"); return }
+        
+        guard let maximumDate = maximumDate
+            else { assertionFailure("Couldn't create maximumDate"); return }
+        
+        datePicker.minimumDate = minimumDate
+        datePicker.maximumDate = maximumDate
+        
+        configureView()
+    }
+    
+    // MARK: - Actions
+    
+    @IBAction func datePickerChanged(_ sender: Any) {
+        
+        let strDate = CustomDateFormatter.default.string(from: datePicker.date)
+        dateTimeLabel.text = strDate
+        
+        let dateComponents = Calendar.current.dateComponents(in: TimeZone(identifier: "UTC")!, from: datePicker.date)
```

Comment:
Can the `TimeZone(identifier: "UTC")!` also be a `private let` or `private lazy var` on the view controller to avoid unnecessary allocation.